users/config.py

```diff
@@ -1 +1,5 @@
 user_status_choices = [("waitlist", "Added to Waitlist"), ("approved", "Approved")]
+auth_type_choices = [
+    ("default", "Default Authentication"),
```

Comment:
I think we can keep default as None right? - we don't need this choice? - and allow it to be nullable?